common/app/common/configuration.scala

```diff
@@ -231,14 +233,10 @@ class GuardianConfiguration extends GuLogging {
     lazy val capiPreviewRoleToAssume: Option[String] =
       configuration.getStringProperty("content.api.preview.roleToAssume")
 
-    lazy val capiPreviewCredentials: AWSCredentialsProvider = {
-      new AWSCredentialsProviderChain(
-        List(
-          Some(new ProfileCredentialsProvider("capi")),
-          capiPreviewRoleToAssume.map(new STSAssumeRoleSessionCredentialsProvider.Builder(_, "capi").build()),
-        ).flatten: _*,
-      )
-    }
+    lazy val capiPreviewCredentials: AWSCredentialsProvider = new AWSCredentialsProviderChain(
+      Seq(new ProfileCredentialsProvider("capi")) ++
+        capiPreviewRoleToAssume.map(new STSAssumeRoleSessionCredentialsProvider.Builder(_, "capi").build()): _*,
+    )
```

Comment:
This is just using slightly cleaner code for constructing the sequence of (1 or 2) `AWSCredentialsProvider`s - no need to make a list of optional things and then `.flatten` it, can just concatenate.